lib/contentful_rails/engine.rb

```diff
@@ -59,9 +59,15 @@ class Engine < ::Rails::Engine
     initializer "add_contentful_mime_type" do
       content_type = "application/vnd.contentful.management.v1+json"
       Mime::Type.register content_type, :contentful_json, [content_type]
-
-      ActionDispatch::ParamsParser::DEFAULT_PARSERS[Mime::Type.lookup(content_type)] = lambda do |body|
-        JSON.parse(body)
+      # Rails 4
+      if ActionDispatch::ParamsParser.constants.include? :DEFAULT_PARSERS
+        ActionDispatch::ParamsParser::DEFAULT_PARSERS[Mime::Type.lookup(content_type)] = lambda do |body|
+          JSON.parse(body)
+        end
+      else # Rails 5
+        ActionDispatch::Http::Parameters::DEFAULT_PARSERS[Mime::Type.lookup(content_type)] = lambda do |body|
+          JSON.parse(body)
+        end
```

Comment:
fyi @prosanelli has a PR open from a couple days ago for this change.   It manually checks the Rails version, but is a little drier :p
https://github.com/weddingwire/contentful_rails/pull/5/files